posts/innovative-snowflake-features.md

```diff
@@ -0,0 +1,208 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Cloud
+- Snowflake
+date: 2019-08-07T10:33:00.000Z
+title: "Innovative Snowflake Features"
+image:
+---
+I recently attended the Snowflake Partner Bootcamp in Baltimore, and was given the opportunity to learn more about Snowflake. In the following blog, I am going to give a run-down of the innovative features Snowflake offers to its clients and some of the benefits of using them.
```

Comment:
Do you have a link to the event you can add?  I like this introduction.

Reply:
Sadly there is no link for it...